variables.tf

```diff
@@ -1,5 +1,56 @@
-#variable "example" {
-#  type        = string
-#  default     = ""
-#  description = "This is just an example of a variable"
-#}
+# Required
+
+variable "tenancy_id" {
+  type = string
+  description = "OCID of the OCI tenancy to be integrated with Lacework"
+}
+
+variable "user_email" {
+  type = string
+  description = "Email associated with the created user"
+}
+
+# Optional
+
+variable "create" {
+  type = bool
+  default = true
+  description = "Set to false to prevent module from creating any resources"
+}
+
+variable "freeform_tags" {
+  type = map
+  default = {}
+  description = "freeform tags for the resources created for Lacework CSPM integration"
+}
+
+variable "name_prefix" {
+	type = string
```

Comment:
indentation. Maybe run https://developer.hashicorp.com/terraform/cli/commands/fmt or add a plugin to your preferred editor.

Reply:
Fixed!